paper-radio-button.html

```diff
@@ -108,13 +116,9 @@
       }
 
       #onRadio {
-        position: absolute;
         box-sizing: content-box;
-        top: 4px;
-        left: 4px;
-        right: 4px;
-        width: 8px;
-        height: 8px;
+        width: 50%;
+        height: 50%;
```

Comment:
It doesn't, since it's no longer absolutely positioned. Instead, it's the only flex child of `#radioContainer` that is positioned using `@apply(--layout-center-center)`. `#offRadio` **is** absolutely positioned, so it can do its own thing while `#onRadio` gets all the centering love.